packages/office-addin-cli/package.json

```diff
@@ -33,7 +33,7 @@
     "rimraf": "^2.7.1",
     "sinon": "^7.5.0",
     "ts-node": "^8.5.2",
-    "tslint": "^5.20.1",
+    "tslint": "^6.1.0",
     "typescript": "^3.7.2"
```

Comment:
As an aside, you may want to see about a separate PR to upgrade the typescript version.

Reply:
I will do this in a separate PR